.gitmodules

```diff
@@ -113,3 +113,6 @@
 [submodule "tools/wgsim/src"]
 	path = tools/wgsim/src
 	url = https://github.com/lh3/wgsim.git
+[submodule "tools/muscle/src"]
```

Comment:
Small change: can you move this up the file so it's right after `modbam2bed`? Just to keep the order sorted by tool name :)

Reply:
ok, fixed now~ I just put it as the last one